neurolang/probabilistic/containment.py

```diff
@@ -40,8 +40,8 @@ def freeze_atom(atom):
     the symbol.
     '''
     args = (
-        Constant(s.name)
-        for s in atom.args
+        Constant(arg.name) if isinstance(arg, Symbol) else arg
+        for arg in atom.args
```

Comment:
Really not sure about this. This code was failing when the atom contained `Constant` terms, as the function assumed all terms to be `Symbol`s